host_core/lib/host_core/vhost/virtual_host.ex

```diff
@@ -427,9 +427,6 @@ defmodule HostCore.Vhost.VirtualHost do
   def handle_cast({:do_stop, _timeout_ms}, state) do
     Logger.debug("Host stop requested manually")
 
-    do_purge(state)
-    publish_host_stopped(state)
-
```

Comment:
We may want to remove the `publish_host_stopped` call, but the `do_purge` is important since it instructs all providers to shutdown with some time to handle cleaning up state and things

Reply:
Ah I see what you mean, that function ends up triggering the process termination. I think I misunderstood the change, and you are likely correct that both of those should be gone. Have you been able to test this either way with something like `nats sub wasmbus.evt.*` to listen for host_stopped events?

Reply:
I've re-added the `do_purge` removal. I checked with `iex -S mix` and `wash ctl stop` listening for the events and the host_stopped event appears only once. In the iex logs the purging works normally too.